src/main/scala-2.11/markets/clearing/engines/CDAMatchingEngine.scala

```diff
@@ -117,7 +117,7 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
         if (residualQuantity < 0) {
           val totalMatch = TotalMatch(bidOrder, incoming, price)
           // add residualOrder back into orderBook!
-          val residualOrder = bidOrder.split(-residualQuantity)
+          val (_, residualOrder) = bidOrder.split(-residualQuantity)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1537948179/issues/source?bid=2921361&fileBranchId=3014278#l120)